### PR TITLE
Mobile App: Migrate to Flutter 2.0

### LINKED ISSUE #19: Spelling error in image picker bottom sheet
**Is your feature request related to a problem? Please describe.**
There is typo in the spelling of **Gallery** in the bottom sheet of image picker.

image

Describe the solution you'd like

**_Option1:_** Make changes in the implementation of the package and then use the modified package.
**_Option2:_** Use an alternative package.

**Are you working on this? (Yes/No)**
Yes

### LINKED ISSUE #77: Upgrade to flutter 2.0
Some deprecations in flutter 2.0 like `FlatButton`, `RaisedButton` etc. needs to be changed to use the new API for the buttons.

The deprecations will be highlighted using pedantic's linting.

### PR DESCRIPTION
**Pull request by manjotsidhu**
Fixes #77, Fixes #19

Describe the changes you have made in this PR -
- [x] Update Packages.
- [x] Updates for [internationalization](https://flutter.dev/docs/development/accessibility-and-localization/internationalization).
- [x] Swap `html_editor` to `flutter_summernote` since `html_editor` is depreciated.
- [x] Flutter Migrations according to their guide.

https://docs.google.com/document/d/1yohSuYrvyya5V1hB6j9pJskavCdVq9sVeTqSoEPsWH0/edit  

**Screenshots of the changes -**
There has been very minor UI changes except for `html_editor` being used for `flutter_summernote` 

`html_editor` -> `flutter_summernote`

### CODE REVIEW

**Comment by manjotsidhu** [comment]
@Nitish145 Null-safety compatible versions do not exist for:
 - oauth2_client
 - flutter_icons
 - flutter_summernote
 - flutter_facebook_login

Warning: Using null-safety features before upgrading all dependencies is
discouraged. For more details see: https://dart.dev/null-safety/migration-guide

We cannot migrate dart to 2.12 (null-safety) because our dependencies are not yet migrated. We have to deal this in another issue.

**Comment by Nitish145** [comment]
Yup, we can do that later. @manjotsidhu

**Review by Nitish145 — APPROVED** [review]

File: lib/ui/views/projects/edit_project_view.dart [hunk 1]
```diff
@@ -106,16 +106,25 @@ class _EditProjectViewState extends State<EditProjectView> {
...
           color: CVTheme.htmlEditorBg,
           border: Border.all(
             color: CVTheme.primaryColorDark,
           ),
         ),
-        value: widget.project.attributes.description ?? '',
         key: _descriptionEditor,
-        height: 300,
+        hasAttachment: true,
+        customToolbar: """
```
[the remark this answers is not in the document]
Comment (manjotsidhu) [reply lib/ui/views/projects/edit_project_view.dart:119]: True, I was thinking to make it a separate component. Shall I do it ?

[the remark this answers is not in the document]
Comment (Nitish145) [reply lib/ui/views/projects/edit_project_view.dart:119]: Yeah, that can be done

**Comment by tachyons** [comment]
👍

**Pull request merged**